ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PostgresStorageBackend.scala

```diff
@@ -332,7 +335,8 @@ object PostgresStorageBackend extends StorageBackend[RawDBBatchPostgreSQLV1] {
       |   typ_in,
       |   configuration_in,
       |   rejection_reason_in
-      | );
+      | )
+      | ON CONFLICT DO NOTHING;
```

Comment:
AFAICS, by moving the conflict detection from `JdbcLedgerDao` to here, you have lost the ability to log warnings for duplicate entries. How about the following changes:
- The above line changes to `preparedStatement.executeUpdate()` (which returns the number of rows affected)
- This method returns the number of rows affected
- The line `rawDBBatchPostgreSQLV1.configurationEntriesBatch.foreach` checks whether the number of affected rows equals the number of configuration entries in the batch. If it differs, it emits a warning.

Reply:
I think it is not really possible to have here a duplication if there is conflict checking in the ledger this regard. I guess the only reason for this is in sandbox-classic: perhaps a race condition issuing the same config change command? :man_shrugging: 

Anyhow I am not entirely sure we need those warnings, and/or it is worth to make these changes just to have those. Let's discuss this offline. If there is a use case I am not against, but I would like to not take unclear complications if we don't have to.
Also we have to be aware that the original indexer code was not caring at all about duplication-s here, and the motivation of this change is to support sandbox-classic. Therefore I think we need to understand whether there is an issue of duplicates in non-sandbox-classic situations as well, because that might changes how we implement this.

Reply:
As discussed with @rautenrieth-da we keep the implementation as is, and we open an implementation to handling duplicate submissions.